app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,45 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @users = [
+      { 'id' => '1', 'username' => 'peter druid', 'number_of_posts' => '3' },
+      { 'id' => '2', 'username' => 'david armstrong', 'number_of_posts' => '1' }
+    ]
```

Comment:
- [ ] It would be better if we retrieve the data from the database through the model instead of hard coding it like we have now. This would be better `@users = User.all`